source/developer/developer-setup.md

```diff
@@ -90,6 +90,14 @@ Any issues? Please let us know on our forums at: http://forum.mattermost.org
 
 ### Archlinux ###
 
+#### Build and install with AUR ####
+
+Archlinux uses a specific feature called [Arch User Repository](https://wiki.archlinux.org/index.php/Arch_User_Repository) to manage extra packages which do not belong to the official repository. It allows you to compile a package from source and then install it via the Arch package manager [pacman](https://wiki.archlinux.org/index.php/Pacman#Additional_commands). This way you will be able to track easily upgrades or dependency issues. This is the recommended way to install extra packages.
+
+There is an AUR package called [mattermost](https://aur.archlinux.org/packages/mattermost) maintained by [mtorromeo](https://github.com/mtorromeo/). Follow the [AUR instructions](https://wiki.archlinux.org/index.php/Arch_User_Repository) to build and install. Please go to the [AUR package page](https://aur.archlinux.org/packages/mattermost/) to leave a comment for sharing feedback or troubleshooting.
```

Comment:
Please add after the first sentence:

`Please note that this package is unofficial since it is built and maintained by the Mattermost community. Follow the ...`